### PR TITLE
Add support for multispeed pumps

### PR DESCRIPTION
**Pull request by jbergler**

### CODE REVIEW

File: src/main.cpp [hunk 1]
```diff
@@ -470,16 +435,26 @@ void mqttCallback(char* topic, byte* payload, unsigned int length) {
     si.setSTMP(int(p.toFloat()*10));
   } else if (property == "heatpump_mode") {
     si.setHPMP(p);
-  } else if (property == "pump1") {
+  } else if (property == "pump1_state") {
```
Comment (wayne-love) [inline src/main.cpp:438]: Why is pump1 being treated different than the other pumps?

File: .vscode/settings.json
Comment (wayne-love) [inline .vscode/settings.json:1]: Do we need to exclude the .vscode from the git repository?

**Review by wayne-love — COMMENTED** [review]
What is the state of this PR, has it been tested?

Additional comments would be good to explain changes.

**Comment by lucasnz** [comment]
This PR does not work with single speed pumps. I have a well implementation, tested against both single and muilti-speed pumps. PR coming shortly.

**Pull request closed without merging**